docs: update nightly repartition docs for SPLIT PARTITION ON COLUMNS (#2561)
GreptimeTeam/docs: docs/user-guide/deployments-administration/manage-data/repartition.md

Otherwise, you can't perform repartitioning.
:::

> GreptimeDB supports repartitioning an unpartitioned table with `PARTITION ON COLUMNS`, and adjusting an existing partitioned table through `SPLIT PARTITION` and `MERGE PARTITION` operations.

Reviewer: ```suggestion
GreptimeDB supports repartitioning an unpartitioned table with `PARTITION ON COLUMNS`, and adjusting an existing partitioned table through repeated `SPLIT PARTITION` and `MERGE PARTITION` operations.
```

> GreptimeDB supports repartitioning an unpartitioned table with `PARTITION ON COLUMNS`, and adjusting an existing partitioned table through `REPARTITION`, `SPLIT PARTITION`, and `MERGE PARTITION` operations.

Reviewer 2: Could we keep this consistent with the reference change? This PR removes `REPARTITION` from the SQL reference syntax, but the guide still lists it as a supported operation here, and the zh guide mirrors the same wording. If the public guidance is to prefer `SPLIT PARTITION`, this should drop `REPARTITION`; otherwise the reference should keep the `REPARTITION` syntax.

## Example

> You can first partition an unpartitioned table, and later keep tuning the layout by repartitioning, merging, or splitting existing partitions.

Reviewer: ```suggestion
You can first partition an unpartitioned table, and later keep tuning the layout by merging existing partitions and splitting them with new rules.
```

);
```

> Adjust an existing partitioned table while keeping the current partition columns:

Reviewer: ```suggestion
Adjust an existing partitioned table. The following example changes the `area` partition key from `South` to `North` for devices with `device_id < 100`:
```
